### ISSUE TITLE
Poor results when autocompleting on Huntingtin gene symbol (HTT)

### ISSUE THREAD

**Issue by kshefchek** [monarch-initiative/monarch-ui#63]
Typing 'HTT' into autocomplete results in hits to URLs.  I think we should adjust the solr schema to omit edge ngrams on IRI fields.  This will require an update to https://github.com/monarch-initiative/golr-schema and the yaml schema.

**Comment by kshefchek** [comment]
See https://github.com/SciGraph/golr-loader/issues/45

**Comment by kshefchek** [comment]
this is fixed, but IRIs are still getting through when a curie prefix does not exist, for example:
"http://uri.neuinfo.org/nif/nifstd/birnlex_18"
"http://identifiers.org/hgnc/6636"

As an aside we have identical HGNC Ids being ingested as distinct nodes/iris
http://identifiers.org/hgnc/6636
https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/6636

**Comment by pnrobinson** [comment]
@kshefchek Can we label this sort of issue 'backend' or 'solrschema' etc?

**Comment by kshefchek** [comment]
Despite my attempts with https://github.com/SciGraph/golr-loader/pull/49 and https://github.com/SciGraph/golr-loader/issues/45, will still have primary IDs from monarch.owl that cannot be resolved to curies with our curie map:

http://www.ncbi.nlm.nih.gov/gene/ - we use https
http://www.ebi.ac.uk/cellline#ND12130 - this doesn't resolve
http://www.informatics.jax.org/marker/MGI:  - we use accession/MGI:

should we filter these out of the search index and track them?

**Comment by monicacecilia** [comment]
For now, it makes sense to filter these out.

**Comment by kshefchek** [comment]
This looks to be fixed, 

![htt](https://user-images.githubusercontent.com/1467299/61582724-544ad800-aae3-11e9-97c4-4d86ae3fc127.png)

**Comment by kshefchek** [comment]
While searching on 'htt' results in HTT gene hits, it looks like we now have IRIs in primary labels on phenotype terms, eg https://beta.monarchinitiative.org/phenotype/ZP:0104672.  I'll make an issue in upheno but wanted to make note of it here.
